src/hermes_claude_agent_sdk/parity/active_suite.py

```
@@ -0,0 +1,1082 @@
+                model=model,
+            )
+
+    if result.classification is ExecutionClassification.ENVIRONMENT_BLOCKED:
+        return _blocked(result.reason_code or "active_environment_blocked")
+    if result.classification is ExecutionClassification.VERIFIED_FAILURE:
+        return _failed(
+            result.reason_code or "active_verified_failure",
+            turn_count=result.turn_count,
+            billing=result.billing,
+        )
+    assert result.evidence_hash is not None and result.state_hash is not None
```

Ruff (0.16.3)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)